CMakeLists.txt

```diff
@@ -308,7 +316,10 @@ if(QUIC_BUILD_TEST)
     set(INSTALL_GTEST OFF CACHE BOOL "Enable installation of googletest. (Projects embedding googletest may want to turn this OFF.)")
     if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
         option(gtest_force_shared_crt "Use shared (DLL) run-time lib even when Google Test is built as static lib." ON)
+    else()
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Why 11, and not 17? Did you do this change before fixing the bad `--std=c++17`?  I wonder if fixing that flag will make this not required.

Reply:
Ah I didn't realize that was a different set of flags. All good, other then as long as it compiles I'd prefer 17 in great to not get any potential incompatibilities. I know there won't be any, but I always like building everything the same way.

Reply:
No complaints there; Will adjust